### ISSUE TITLE
Refactor Route&Lines Routes list out of the table structure

### ISSUE THREAD

**Issue by niknii-vincit** [HSLdevcom/jore4#1544]
The list of routes and their stops should be refactored away from the table structure, because it's not accessible.

![Image](https://github.com/HSLdevcom/jore4/assets/135983800/a43944e0-5f41-4709-bff4-399a88832a92)

The view just happens to be similar to a table structure, with a heading row and data rows, but in fact it's not a table. This might look fine, but it makes screen-readers behave in an unintuitive way.

Additionally, this structure blocks a proper way to implement the `aria-controls` attribute, because there's not parent div for the table data rows.

The `aria-controls` should be implemented somehow like this:
```html
...
<div id="expandableRow">
   <h3>Title</h3>
   <!-- valuable data snippets, or "heading row" -->
   <button onClick="expandRow" aria-controls="allTheData" /> <!-- aria-controls is here and refers to the data element below-->
   <div id="allTheData" display=none>
      <!-- many -->
      <!-- rows -->
      <!-- of data -->
   </div>
</div>
...
```

The ticket #1440 added `controls` attribute to `AccordionButton` element (which makes it into `aria-controls`), but as the structure was a table then it couldn't be properly implemented, and nothing is more permanent than a temporary "fix"...
